runner_test.go

```diff
@@ -25,7 +26,7 @@ func TestScenario(t *testing.T) {
 	require.NoError(t, os.MkdirAll(execDir, 0755))
 	require.NoError(t, clearDir(execDir))
 
-	for _, file := range files {
+	for _, file := range filteredFiles(t, files) {
```

Comment:
Something like this is already built into `go test`, use the `-run` flag.
```bash
$ go test ./ -run TestScenario/3_struct_mapping_extend.yml
--- FAIL: TestScenario (0.04s)
    --- FAIL: TestScenario/3_struct_mapping_extend.yml (0.04s)
         ....
```

Reply:
Duh! I just need to configure my editor (Goland) correctly.

Reverted.